core/Network/TLS/Extra/Cipher.hs

```diff
@@ -652,6 +655,28 @@ cipher_ECDHE_RSA_AES256GCM_SHA384 = Cipher
     , cipherMinVer       = Just TLS12 -- RFC 5289
     }
 
+cipher_AES128GCM_SHA256 :: Cipher
+cipher_AES128GCM_SHA256 = Cipher
+    { cipherID           = 0x1301
+    , cipherName         = "AES128GCM-SHA256"
+    , cipherBulk         = bulk_aes128gcm
+    , cipherHash         = SHA256
+    , cipherPRFHash      = Nothing
+    , cipherKeyExchange  = CipherKeyExchange_TLS13
+    , cipherMinVer       = Just TLS13ID16
+    }
+
+cipher_AES256GCM_SHA384 :: Cipher
+cipher_AES256GCM_SHA384 = Cipher
+    { cipherID           = 0x1302
+    , cipherName         = "AES256GCM-SH384"
+    , cipherBulk         = bulk_aes256gcm
+    , cipherHash         = SHA384
+    , cipherPRFHash      = Nothing
```

Comment:
I see that TLS13 replaces the HMAC PRF with HKDF.  Is the separate PRFHash field now redundant? Perhaps it always the case that there's only one hash per cipher, that's used both for HKDF and the finished HMAC?  In that case, my comment about PRFHash above can be ignored.